multicluster/test/e2e/service_test.go

```diff
@@ -76,9 +93,19 @@ func (data *MCTestData) setUpServiceExport(t *testing.T) {
 	time.Sleep(importServiceDelay)
 }
 
+// Deploy ServiceExports in east and west clusters
+func (data *MCTestData) setUpServiceExport1(t *testing.T) {
```

Comment:
I don't think you need a new ServiceExport. ServiceExport are already created in both east and west clusters. You just need to create a client pod in the west cluster and verify the connection in the same cluster.
Btw, it's not proper way to create a method named `setUpServiceExport1`, if you do need similar function, you may add some parameters to reduce code duplication.